suitcase/tiff_series/tests/tests.py

```diff
@@ -113,3 +113,28 @@ def test_export(tmp_path, example_data):
         actual = tifffile.imread(str(filename))
         streamname = os.path.basename(filename).split('-')[0]
         assert_array_equal(actual, expected[streamname])
+
+
+@pytest.mark.parametrize("directory", ['', '/test-', '/scan_{start[uid]}-'])
+def test_directory_formatting(directory, example_data, tmp_path):
+    ''' runs a test of the file_prefix formatting.
+
+    ..note::
+
+        Due to the `file_prefix_list` and `example_data` `pytest.fixture`'s
+        this will run multiple tests each with a range of file_prefixes,
+        detectors and event_types. see `suitcase.utils.conftest` for more info.
+
+    '''
+    collector = example_data()
+    directory = str(tmp_path)+directory
```

Comment:
Will this be better?
```suggestion
    directory = f'{tmp_path}{directory}'
```

Reply:
I think this will cause issues, as `directory` is a string that potentially needs formatting

Reply:
I think it's identical:
```python
In [1]: directory = '/tmp/my/dir'

In [2]: tmp_path = '/tmp'

In [3]: str(tmp_path)+directory
Out[3]: '/tmp/tmp/my/dir'

In [4]: f'{tmp_path}{directory}'
Out[4]: '/tmp/tmp/my/dir'
```

Reply:
the issue here is that directory is not a simple string, it is instead something like:
`directory = '/{uid}'`
and I don't think
`f'{tmp_path}{directory}'.format(uid=12345)`
will work as it is mixing formating approaches

Reply:
I think we are fine here:
```py
In [1]: directory = '/{uid}'

In [2]: tmp_path = '/my/tmp/dir'

In [3]: str(tmp_path)+directory
Out[3]: '/my/tmp/dir/{uid}'

In [4]: f'{tmp_path}{directory}'
Out[4]: '/my/tmp/dir/{uid}'

In [5]: (str(tmp_path)+directory).format(uid=12345)
Out[5]: '/my/tmp/dir/12345'

In [6]: f'{tmp_path}{directory}'.format(uid=12345)
Out[6]: '/my/tmp/dir/12345'
```

Reply:
as I mentioned above, this method _fails_ when I implement in the repo code even though the simple code, as you produced here, works fine. I still think it is best _not_ to mix the formatting apprroaches.

Reply:
OK. Initially I was about to propose adding blank spaces around `=`, but then thought it may be more readable with f-strings. I don't have strong preference here, so either way will work.

Reply:
I updated the suggestion to sum two strings with PEP8 compatibility.

Reply:
And the failed case in the repo, which you mention, may happen due to operations with `Path` objects or the manager type of objects, not pure strings. Is it where you saw failures?